docs/limitations.md

```diff
@@ -155,7 +155,7 @@ with expansion:
 
 
 ### Clang/G++ -- skipping leaf sections after an exception
-Some versions of `libc++` and `libstdc++` (or their runtimes) have a bug with `std::uncaught_exception()` getting stuck returning `true` after rethrow, even if there are no active exceptions. One such case is this snippet, which skipped the sections "a" and "b", when compiled against `libcxxrt` from master
+Some versions of `libc++` and `libstdc++` (or their runtimes) have a bug with `std::uncaught_exception()` getting stuck returning `true` after rethrow, even if there are no active exceptions. One such case is this snippet, which skipped the sections "a" and "b", when compiled against `libcxxrt` from devel branch
```

Comment:
Assuming I found [the right repo](https://github.com/libcxxrt/libcxxrt), they still use `master` for their main branch. Changing the name is wrong here.

Reply:
Sorry, I misunderstood it. I will revert it. Thanks for clarifying

Reply:
> Feel free to use "from the master branch" if you think it makes it clearer though.

Done